tests/puppeteer/pages/BO/design/emailThemes/index.js

```diff
@@ -0,0 +1,45 @@
+require('module-alias/register');
+const BOBasePage = require('@pages/BO/BObasePage');
+
+module.exports = class EmailThemes extends BOBasePage {
+  constructor(page) {
+    super(page);
+
+    this.pageTitle = 'Email Theme •';
+
+    // Selectors
+    this.blockDiv = 'div.content-div div.justify-content-center:nth-child(%ID)';
+    // Email Theme table selectors
+    this.emailThemeTable = `${this.blockDiv.replace('%ID', 5)} table.grid-table`;
```

Comment:
in this case there is only one table, so `table.grid-table` should do it.
but it would be better to have id on the block or a name like in the bloc translate emails `form[name='generate_mails']`.
do you think i should delete `${this.blockDiv.replace('%ID', 5)}` ?

Reply:
Indeed it's too bad there is no ID to target this specific table, so if we add one it won't work any more
But for now yes I think you should only use `table.grid-table` to select it, because the chances are high that we add/remove/switch a block in this page The chances of a new table are low, but when it happens we will add the appropriate IDs

Reply:
done